src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -934,4 +935,44 @@ public static Vec3d rayTraceEyeHitVec(EntityLivingBase entity, double length)
         RayTraceResult git = rayTraceEyes(entity, length);
         return git == null ? null : git.hitVec;
     }
+
+    public static boolean onInteractEntityAt(EntityPlayer player, Entity entity, RayTraceResult ray, ItemStack stack, EnumHand hand)
```

Comment:
No one has established a convention really for which goes where, I'm just going to leave them here until lex tells me to move them :P`